action.yaml

```
@@ -64,7 +64,7 @@ runs:
       run: |
         # For bats-assert and friends
         brew tap kaos/shell
-        brew install bats-core bats-assert bats-support jq mkcert yq
+        brew install bats-file bats-assert bats-support jq mkcert yq
```

```suggestion
        brew install bats-core bats-file bats-assert bats-support jq mkcert yq
```

I don't think you're correct. There are SO MANY THINGS in SO MANY PLACES here.

* https://github.com/bats-core/bats-core
* Then the docs take you to https://bats-core.readthedocs.io/en/stable/faq.html#how-can-i-use-helper-libraries-like-bats-assert
* Which takes you to https://github.com/ztombol/bats-docs
* Which tells you to install those 3 *libraries*, bats-support, bats-assert, and bats-file

I find it mind boggling. And then the brew tap is kaos? ???

---

Yes, this is not clear at all. 
And if you read here: https://github.com/ztombol/bats-docs#homebrew

> Note: The required dependencies, bats-support as well as bats from the core tap will be installed automatically for you, with any of the two previous commands.

suggesting that we don't have to specify "brew install bats-support" ... 

To keep it simple, I will do as you suggested in your review and run ALL install commands 
`brew install bats-core bats-file bats-assert bats-support jq mkcert yq`

It shouldn't do any harm.

Thanks

---

I'm baffled about all this, but have been for some time. But now the thing about bats-support makes you wonder if there's been some drama, etc.